src/main/java/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,21 @@
+package strategy;
+
+import java.util.Objects;
+import model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public int getQuantityToCalculate(FruitTransaction fruitTransaction) {
+        return fruitTransaction.getQuantity();
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        return obj instanceof BalanceOperation;
+    }
```

Comment:
Do you really need equals and hashCode here? If you add it for testing better just compare expected and actual using the method `getClass()`